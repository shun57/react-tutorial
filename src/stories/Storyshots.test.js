import initStoryshots from '@storybook/addon-storyshots';
import { createSerializer } from '@emotion/jest';
initStoryshots({
  snapshotSerializers: [createSerializer()],
});