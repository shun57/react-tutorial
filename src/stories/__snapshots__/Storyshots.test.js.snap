// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots SquareGame/Bord Default 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
    <button
      className="square"
      onClick={[Function]}
    />
  </div>
</div>
`;

exports[`Storyshots SquareGame/Bord Default And Triangle 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      ○
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ○
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ○
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      ×
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ×
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ×
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
</div>
`;

exports[`Storyshots SquareGame/Bord Star 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      ☆
    </button>
  </div>
</div>
`;

exports[`Storyshots SquareGame/Bord Triangle 1`] = `
<div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
  <div
    className="board-row"
  >
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
    <button
      className="square"
      onClick={[Function]}
    >
      △
    </button>
  </div>
</div>
`;

exports[`Storyshots SquareGame/Game Default 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
      <div
        className="board-row"
      >
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
        <button
          className="square"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
  <div
    className="game-info"
  >
    <div>
      次のプレイヤー: X
    </div>
    <ol>
      <li>
        <button
          onClick={[Function]}
        >
          Go to game start
        </button>
      </li>
    </ol>
  </div>
</div>
`;

exports[`Storyshots SquareGame/Square Default 1`] = `
<button
  className="square"
/>
`;

exports[`Storyshots SquareGame/Square Star 1`] = `
<button
  className="square"
>
  ☆
</button>
`;

exports[`Storyshots SquareGame/Square Triangle 1`] = `
<button
  className="square"
>
  △
</button>
`;
